src/sentry/hybridcloud/rpc/filter_query.py

```diff
@@ -132,4 +132,5 @@ def get_many(self, filter: FILTER_ARGS) -> list[RPC_RESPONSE]:
         return [self.serialize_rpc(o) for o in self.query_many(filter=filter)]
 
     def get_many_ids(self, filter: FILTER_ARGS) -> list[int]:
-        return [o.id for o in self.query_many(filter=filter, select_related=False)]
+        # Using getattr to avoid mypy errors that Model does not have attr id
+        return [getattr(o, "id") for o in self.query_many(filter=filter, select_related=False)]
```

Comment:
this should be `bound=Model` but our base `Model` in `sentry.db.models.base` (it has `id: Field[int, int]`)

Reply:
I tried that. However, the models in `socialauth` don't inherit from `BaseModel` so type errors show up in different parts of the codebase.

Reply:
I've been fixing those when running into them -- shouldn't be too bad to move them over (yeah I know they are vendored modules, but we've already modified them so much that we're never going to be able to cleanly pull or unfork)